### PR TITLE
LEARNER-4122 Android - Verify 'Accessibility ID' is properly set against each element - for Whats New Screen

### PR DESCRIPTION
**Pull request by nedo**
LEARNER-4122 Android - Verify 'Accessibility ID' is properly set against each element - for Whats New Screen

* all element locations have proper locators based on given ids

### CODE REVIEW

**Comment by nedo** [comment]
@raeeschachar PR is ready for review

cc @staubina

**Comment by nedo** [comment]
@staubina I've rebased this PR, please have a look and comment for further. 


Thanks,

File: common/globals.py [hunk 1]
```diff
@@ -254,6 +252,15 @@ def __init__(self, target_activity, logs):
         self.driver = None
 
     def __call__(self, driver):
+        """
+        Arguments:
+                driver (webdriver): webdriver instance variable
```
Comment (staubina) [inline common/globals.py]: Minor nit: Spacing on the comment.

Comment (nedo) [reply common/globals.py]: done

**Comment by nedo** [comment]
Merged with master through command line. Closing this PR

**Pull request closed without merging**